src/rgw/driver/rados/rgw_cr_rados.cc

```diff
@@ -717,13 +717,13 @@ int RGWRadosBILogTrimCR::request_complete()
   return r;
 }
 
-int send_sync_notification(const DoutPrefixProvider* dpp,
-                           rgw::sal::RadosStore* store,
-                           rgw::sal::Bucket* bucket,
-                           rgw::sal::Object* obj,
-                           const rgw::sal::Attrs& attrs,
-                           uint64_t obj_size,
-                           const rgw::notify::EventTypeList& event_types) {
+void send_sync_notification(const DoutPrefixProvider *dpp,
```

Comment:
"rgw::sal::RadosStore *store" (and similar) are not preferred to "rgw::sal::RadosStore* store", source Stroustrup, "The C++ Programming Language"--because the decl is of a "pointer to rgw::sal::RadosStore" named store--it does not make sense to link the pointer-deriving symbol to the name being declared, it goes with the stem type.

Reply:
(so please don't change these)

Reply:
done